sql_metadb/derived_tables/po_line_fund_distribution_transactions.sql

```diff
@@ -0,0 +1,166 @@
+        poline_number;
+
+CREATE INDEX ON folio_derived.po_line_fund_distribution_transactions (po_id);
+
+CREATE INDEX ON folio_derived.po_line_fund_distribution_transactions (po_number);
+
+CREATE INDEX ON folio_derived.po_line_fund_distribution_transactions (po_workflowStatus);
+
+CREATE INDEX ON folio_derived.po_line_fund_distribution_transactions (po_line_id);
+
+CREATE INDEX ON folio_derived.po_line_fund_distribution_transactions (poline_number);
+
+CREATE INDEX ON folio_derived.po_line_fund_distribution_transactions (poline_title_or_package);
+
+CREATE INDEX ON folio_derived.po_line_fund_distribution_transactions (poline_listUnitPrice);
+
+CREATE INDEX ON folio_derived.po_line_fund_distribution_transactions (poline_quantityPhysical);
+
+CREATE INDEX ON folio_derived.po_line_fund_distribution_transactions (poline_listUnitPriceElectronic);
+
+CREATE INDEX ON folio_derived.po_line_fund_distribution_transactions (poline_quantityElectronic);
+
+CREATE INDEX ON folio_derived.po_line_fund_distribution_transactions (poline_currency);
+
+CREATE INDEX ON folio_derived.po_line_fund_distribution_transactions (poline_discount);
+
+CREATE INDEX ON folio_derived.po_line_fund_distribution_transactions (poline_discountType);
+
+CREATE INDEX ON folio_derived.po_line_fund_distribution_transactions (poline_additionalCost);
+
+CREATE INDEX ON folio_derived.po_line_fund_distribution_transactions (poline_estimated_price);
+
+CREATE INDEX ON folio_derived.po_line_fund_distribution_transactions (po_line_fyroAdjustmentAmount);
+
+CREATE INDEX ON folio_derived.po_line_fund_distribution_transactions (fund_distribution_value);
+
+CREATE INDEX ON folio_derived.po_line_fund_distribution_transactions (fund_distribution_type);
+
+CREATE INDEX ON folio_derived.po_line_fund_distribution_transactions (budget_id);
+
+CREATE INDEX ON folio_derived.po_line_fund_distribution_transactions (budget_name);
+
+CREATE INDEX ON folio_derived.po_line_fund_distribution_transactions (fund_id);
+
+CREATE INDEX ON folio_derived.po_line_fund_distribution_transactions (fund_code);
+
+CREATE INDEX ON folio_derived.po_line_fund_distribution_transactions (fiscal_year_id);
+
+CREATE INDEX ON folio_derived.po_line_fund_distribution_transactions (fiscal_year);
+
+CREATE INDEX ON folio_derived.po_line_fund_distribution_transactions (transaction_id);
+
+CREATE INDEX ON folio_derived.po_line_fund_distribution_transactions (transaction_currency);
+
+CREATE INDEX ON folio_derived.po_line_fund_distribution_transactions (transaction_amount);
+
+CREATE INDEX ON folio_derived.po_line_fund_distribution_transactions (transaction_encumbrance_initial_amount);
+
+CREATE INDEX ON folio_derived.po_line_fund_distribution_transactions (transaction_encumbrance_amount_expended);
+
+CREATE INDEX ON folio_derived.po_line_fund_distribution_transactions (transaction_encumbrance_order_type);
+
+CREATE INDEX ON folio_derived.po_line_fund_distribution_transactions (transaction_encumbrance_subscription);
+
```

Comment:
As we very recently agreed we should add VACUUM ANALYZE to all derived tables., which is missing here.